runtime-modules/versioned-store/src/lib.rs

```diff
@@ -697,7 +697,9 @@ impl<T: Trait> Module<T> {
         }
     }
 
+    // Preserve special formatting.
     #[rustfmt::skip]
+    #[allow(clippy::match_like_matches_macro)]
```

Comment:
Seems to me that we should drop these deprecated modules especially when we launch a new chain for olympia.

Reply:
Yes. You are correct. I was going to drop these modules during the integration of multiple modules in the olympia.

Reply:
Okay sounds like a good plan.